src/angular-bootstrap-toggle.js

```diff
@@ -1,289 +1,313 @@
-                      if (ngModelCtrl.$viewValue) {
-                        angular.element(self.wrapperElement).
-                          removeClass('off ' + self.offClass).
-                          addClass(self.onClass);
-                      } else {
-                        angular.element(self.wrapperElement).
-                          addClass('off ' + self.offClass).
-                          removeClass(self.onClass);
-                      }
-                    };
-
-                    $scope.onSwitch = function (evt) {
-                       if (self.disabled) {    // prevent changing .$viewValue if .disabled == true
-                            return false;
-                       } else {
-                           ngModelCtrl.$setViewValue(!ngModelCtrl.$viewValue);
-                           ngModelCtrl.$render();
-                       }
-                       return true;
-                    };
-
-                    angular.forEach( toggleConfigKeys, function (k, i) {
-                        $attrs.$observe(k, function (v) {
-                            if (self[k] !== v) {
-                                self[k] = v;
-                                self.computeStyle();
-                            }
-                        });
-                    });
-
-                }])
-
-        .directive('toggle', function () {
-                return {
-                    restrict: 'E',
-                    template: '<div ng-cloak class="toggle btn off" ng-style="wrapperStyle"' +
-                                'ng-click="onSwitch($event)">' +
-                                '<div class="toggle-group">' +
-                                '<label class="btn toggle-on-pad"></label>' +
-                                '<label class="btn toggle-off-pad active"></label>' +
-                                '<span class="btn btn-default toggle-handle"></span>' +
-                               '</div>' +
-                              '</div>',
-                    scope: {
-                        ngModel: '='
-                    },
-                    require: ['toggle', 'ngModel'],
-                    controller: 'ToggleController',
-                    controllerAs: 'toggle',
-                    compile: function (element, attrs, transclude) {
-                        return {
-                            post: function (scope, element, attrs, ctrls) {
-                                var toggleCtrl = ctrls[0], ngModelCtrl = ctrls[1];
-                                toggleCtrl.element = element;
-                                toggleCtrl.init(ngModelCtrl);
-                            },
-                            pre: function () {}
-                        };
-                    }
-                };
-            }
-        );
+				}
+			])
+		.directive('toggle',
+			function () {
+				return {
+					restrict: 'E',
+					template: "<div ng-cloak class=\"toggle btn off\" ng-style=\"wrapperStyle\"" +
```

Comment:
i have escaped the double quoted strings because VScode was not stop complaining about it.

Reply:
Try Atom then ;-)